schedule/yam/agama_raid_unattended_use_existing_md_raid.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_raid_unattended_change_disk
```

Comment:
Normally here will set the same name as the file: 
  agama_raid_unattended_use_existing_md_raid

Reply:
Yeah I forgot to update thanks